pkg/monitor/format/format.go

```diff
@@ -249,6 +251,9 @@ func (m *MonitorFormatter) FormatSample(data []byte, cpu int) {
 		m.policyVerdictEvents(prefix, data)
 	case monitorAPI.MessageTypeRecCapture:
 		m.recorderCaptureEvents(prefix, data)
+	case monitorAPI.MessageTypeTraceSock:
+		log.Infof("sock-traces-debug %+v %v", data, messageType)
```

Comment:
This should not print frequent events to the log at `Info` level.

Reply:
This was just a stub commit, I've removed it. Sebastian will be adding a parser to dump the data in follow-ups.